configure.ac

```diff
@@ -141,6 +141,12 @@ AC_ARG_WITH([ipset],
        [AC_PATH_PROG([IPSET], [ipset], [/bin/false], [$FW_TOOLS_PATH])])
 AC_SUBST(IPSET)
 
+AC_ARG_ENABLE([log-target-syslog],
+       [AS_HELP_STRING([--enable-log-target-syslog], [Write log messages to syslog only])],
+       [ENABLE_LOG_TARGET_SYSLOG="$enableval"], [ENABLE_LOG_TARGET_SYSLOG='no'])
+AM_CONDITIONAL(ENABLE_LOG_TARGET_SYSLOG, [test x$ENABLE_LOG_TARGET_SYSLOG = xyes])
+AC_SUBST(ENABLE_LOG_TARGET_SYSLOG)
+
```

Comment:
AFAICS, this is setting the default log target. Does it makes sense to call it `default-log-target` and let values be; mixed, file, syslog. Default would be mixed. This makes it more general and not a syslog specific build option.

Then you don't even need `OPTARG_LOG_TARGET`. Just use the `DEFAULT_LOG_TARGET` value as set by autoconf. This also lets distributions change the built in default.

Reply:
Looks great. Thank you.

Reply:
Thanks. Merging after CI.